src/service.js

```diff
@@ -612,6 +612,10 @@ module.exports = function (mixinOptions) {
 				if (!this.shouldUpdateGraphqlSchema && this.graphqlHandler) {
 					return;
 				}
+				
+				if (this.apolloServer) {
+					this.apolloServer.stop();
```

Comment:
I'm concerned here as Apollo's docs specify this as an `async` method but there is no promise handling on this call.

https://www.apollographql.com/docs/apollo-server/api/apollo-server/#stop

Reply:
Shared concerns.
The stop function will gracefully close the subscription server and so the socket and if you're unlucky the new sever will try listen before the old one is fully closed
(https://github.com/apollographql/apollo-server/blob/b7a91df76acef748488eedcfe998917173cff142/packages/apollo-server-core/src/ApolloServer.ts#L594)

Reply:
fixed